alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -0,0 +1,83 @@
+# PrometheusDuplicateTimestamps
+
+## Meaning
+
+The `PrometheusDuplicateTimestamps` alert is triggered when there is a constant
+increase in dropped samples due to them having identical timestamps.
+
+## Impact
+
+Unwanted samples might be scraped, and desired samples might be dropped.
+Consequently, queries may yield incorrect and unexpected results.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+   for default cluster monitoring or for the instance that monitors user-defined
+   projects by viewing the alert message's `namespace` label: the namespace for
+   default cluster monitoring is `openshift-monitoring` and, the namespace for
+   user workload monitoring is `openshift-user-workload-monitoring`.
+
+2. Review the logs for the affected Prometheus instance:
+
+   ```shell
+   $ NAMESPACE='<value of namespace label from alert>'
+
+   $ oc -n $NAMESPACE logs -l 'app.kubernetes.io/name=prometheus' | \
+   grep 'Error on ingesting samples with different value but same timestamp.*' \
+   | sort | uniq -c | sort -n
+   level=warn ... scrape_pool="the-scrape-pool" target="an-involved-target" \
+   msg="Error on ingesting samples with different value but same timestamp"
+   ```
+
+   Warning logs similar to the one above should be present.
+
+   In the case where targets are defined via a `ServiceMonitor` or `PodMonitor`,
+   the `scrape_pool` label will be in the format
+   `<namespace>/<service-monitor-name>/<endpoint-id>` or
```

Comment:
the format is `serviceMonitor/<namespace>/...` or  `podMonitor/<namespace>/...`

Reply:
another copy/past :) otherwise we'll have naming conflicts, I do remember having to deal with a similar issue though...